src/manpage.rs

```diff
@@ -0,0 +1,484 @@
+    match item {
+        HelpItem::Decor { help } => {
+            manpage.subsection(help);
+        }
+        HelpItem::BlankDecor => {
+            manpage.text([]);
+        }
+        HelpItem::Positional {
+            strict: _,
+            metavar,
+            help,
+        } => {
+            manpage.label(|l| {
+                l.metavar(metavar.0);
+            });
+            if let Some(help) = help {
+                manpage.text([norm(help)]);
+            }
+        }
+        HelpItem::Command {
+            name,
+            short: _,
+            help,
+            meta: _,
+            info: _,
+        } => {
+            if let Some(path) = command_path {
+                manpage.label(|l| {
+                    l.bold(path).space().bold(name);
+                });
+                if let Some(help) = help {
+                    manpage.text([norm(help)]);
+                }
+            }
+        }
+
+        HelpItem::Flag { name, help } => {
+            manpage.label(|l| {
+                l.shortlong(name);
+            });
+            if let Some(help) = help {
+                manpage.text([norm(help)]);
+            }
+        } // todo!(),
+        HelpItem::Argument {
+            name,
+            metavar: mvar,
+            env,
+            help,
+        } => {
+            manpage.label(|l| {
+                l.shortlong(name).norm("=").metavar(mvar.0);
+                if let Some(env) = env {
+                    l.env(env);
+                }
+            });
+
+            if let Some(help) = help {
+                manpage.text([norm(help)]);
+            }
+        } // todo!(),
+    }
+}
+
+fn norm<S: Into<String>>(s: S) -> Inline {
+    Inline::Roman(s.into())
+}
+
+fn bold<S: Into<String>>(s: S) -> Inline {
+    Inline::Bold(s.into())
+}
+
+fn italic<S: Into<String>>(s: S) -> Inline {
+    Inline::Italic(s.into())
+}
+
+fn newline() -> Inline {
+    Inline::LineBreak
+}
+
+impl<T> OptionParser<T> {
+    /// Render `OptionParser` as a [man page](https://en.wikipedia.org/wiki/Man_page)
+    ///
+    /// - `date` - date to display at the end of te man page, free form
+    /// - `app` - application name
+    ///
+    /// # Usage
+    /// You can generate a test file as a part of your test suite:
+    /// ```no_run
+    /// # use bpaf::*;
+    /// #[derive(Debug, Clone, Bpaf)]
+    /// #[bpaf(options)]
+    /// /// Program that performs an operation
+    /// struct Options {
+    ///     flag: bool,
+    /// }
+    ///
+    /// #[test]
+    /// fn update_test_file() {
+    ///     let manpage = options().as_manpage("sample", Section::General, "May 2020");
+    ///     std::fs::write("sample.1", manpage).expect("Unable to save manpage file");
+    /// }
+    /// ```
+    ///
+    /// Requires `manpage` feature which is disabled by default.
+    pub fn as_manpage(&self, app: &str, section: Section, date: &str) -> String {
```

Comment:
This does not follow the [Rust API guidelines for an `as_` functions](https://rust-lang.github.io/api-guidelines/naming.html#c-conv).  This should instead be a `to_` function.

Reply:
Yea, will have to fix it, thank you for pointing this out.